python-dependencies/production-dependencies.md

```diff
@@ -0,0 +1,27 @@
+---
+description: Aufzählung und Erläuterung der benutzten Pythonpakete
+---
+
+# Production Dependencies
+
+In der Production Version werden 15 Pythonpakete genutzt, welche im Folgenden aufgelistet und kurz erläutert
+werden.
+
+
+| Paketname         | Homepage          | Erläuterung           |
+|-------------------|-------------------|-----------------------|
+| django            | [djangoproject](https://www.djangoproject.com/) | Basis Webframework des Projektes    |
+| gunicorn          | [gunicorn](https://gunicorn.org/)| WSGI HTTPS Server  |
+| gevent            | [gevent](http://www.gevent.org/) |                    |
```

Comment:
Ehrlich gesagt.. keine Ahnung. Irgendwie war das immer in meinen Dependencies drin. In der `gunicorn` Dokumentation steht, dass man es als Worker nutzen kann. Wir könnten einfach mal probieren das Projekt ohne `gevent` aufzusetzen. Eventuell wird es auch automatisch durch `gunicorn` runter geladen?